.github/workflows/pull-request.yaml

```diff
@@ -51,30 +51,50 @@ jobs:
       with:
         go-version-file: go.mod
 
+    - uses: actions/cache@v4
+      with:
+        path: |
+          build/.melange-cache
+        key: melange-cache
+
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v3
 
-    - name: Login to Registry
-      uses: docker/login-action@v3
-      with:
-        registry: registry.staging.replicated.com
-        username: ${{ secrets.REGISTRY_USERNAME_STAGING }}
-        password: ${{ secrets.REGISTRY_PASSWORD_STAGING }}
+    - name: Setup Melange
+      uses: chainguard-dev/actions/setup-melange@main
 
     - name: Build and push local-artifact-mirror image
       run: |
         SHA=${{ github.event.pull_request.head.sha }}
         export SHORT_SHA=dev-${SHA::7}
-        export LOCAL_ARTIFACT_MIRROR_IMAGE=registry.staging.replicated.com/library/embedded-cluster-local-artifact-mirror
+        export LOCAL_ARTIFACT_MIRROR_IMAGE=replicated/embedded-cluster-local-artifact-mirror-staging
```

Comment:
nit: why not make those inline with the `make apko` command similar to the make commands below it?

Reply:
it would be 4 lines of code i would have to duplicate 3 times

Reply:
oh it's used for the other make commands as well. nvm then